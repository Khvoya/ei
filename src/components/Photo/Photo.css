.Photo {
    display: flex;
    flex-direction: column;
    justify-content: center;
}
.Photo h1 {
    width: 100%;
    opacity: 0.96;
    color: #000000;
    font-family: Roboto;
    font-size: 60px;
    font-weight: 400;
    line-height: 73.79px;
    margin-bottom: 72px;
}
.Photo__wrap {
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    box-sizing: border-box;
    justify-content: center;
    width: 100%;
    height: 614px;
    padding: 0 11.7%;
}