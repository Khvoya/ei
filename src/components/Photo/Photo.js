import React, { Component } from "react";
import {PHOTOS} from "../../constants/photos";
import PhotoItem from './PhotoItem/PhotoItem';
import "./Photo.css";

class Photo extends Component {
    render() {
        const photoItems = PHOTOS.map(item => <PhotoItem key={item.photo} photo={item.photo}/>);
        return (
            <div id="photo" className="Photo">
                <h1>Photo</h1>
                <div className="Photo__wrap">
                    {photoItems}
                </div>
            </div>
        );
    }
}

export default Photo;
