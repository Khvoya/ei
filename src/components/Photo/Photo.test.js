import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Photo from './Photo';

describe('Photo', () => {

  it('should render correctly', () => {
    const output = shallow(
      <Photo />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
