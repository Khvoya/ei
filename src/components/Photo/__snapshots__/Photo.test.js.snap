// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
